nifi-nar-bundles/nifi-websocket-bundle/nifi-websocket-services-jetty/src/main/java/org/apache/nifi/websocket/jetty/JettyWebSocketClient.java

```diff
@@ -293,35 +303,37 @@ private void connect(final String clientId, String sessionId) throws IOException
             try {
                 router = routers.getRouterOrFail(clientId);
             } catch (WebSocketConfigurationException e) {
-                throw new IllegalStateException("Failed to get router due to: "  + e, e);
+                throw new IllegalStateException("Failed to get router due to: " + e, e);
```

Comment:
Should this message call `e.getMessage()`?
```suggestion
                throw new IllegalStateException("Failed to get router due to: " + e.getMessage(), e);
```